Segmentation of flight HALO-20240827a

Lucalino opened the pull request:
Some weird behaviour of Meteor overpass calculation still needs to be understood (perhaps together during review) but otherwise, this segmentation is ready for review.

tmieslinger commented on scripts/segmentation_HALO-20240827a.md:
```diff
@@ -0,0 +1,562 @@
+    "leg 1 overpassing METEOR",
+)
+
+seg17 = (
+    slice("2024-08-27T17:03:16", "2024-08-27T17:06:15"),
+    ["kind"],
+    "steep turn",
+    ["steep turn with constant roll angle, perhaps worth a new segment type `turn`?"]
+)
```
I would not make this and similar patterns a segment as I cannot think of any use case for analysing it.

tmieslinger reviewed (approved):
Thanks for segmenting this challenging flight!! :tada:
I added the METEOR events and comments in the respective overpass segments.

Lucalino commented:
Sorry I removed your comment on the steep turns @tmieslinger, I actually just wanted to remove my response as it showed up twice.. Anyway: I removed the short segments in which the aircraft turned sharply as I agree that there isn't any clear scientific use for these segments.
